pkg/api/schema/docs.md

```diff
@@ -3,147 +3,149 @@
 
 ## Table of Contents
 
-- [schema.proto](#schema.proto)
```

Comment:
This looks like some protobuf generator version mismatch - that doc itself seems to be OK - it's only internal anchor names that changed, however it produces a large diff. Can you check if there's some trivial way to avoid this?